src/vs/workbench/services/search/test/common/replace.test.ts

```diff
@@ -68,9 +68,10 @@ suite('Replace Pattern test', () => {
 		// $$0 => no treatment
 		testParse('hello$$0', 'hello$$0', false);
 
-		// $0 => $&
-		testParse('hello$0', 'hello$&', true);
-		testParse('hello$02', 'hello$&2', true);
+		// $0 => $0
+		// No change after #149990
+		testParse('hello$0', 'hello$0', true);
```

Comment:
These can be deleted too

Reply:
Done.  Deleted the other tests which referenced $0.